clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/StorageService.java

```diff
@@ -409,6 +414,20 @@ public synchronized void removeStorageEngine(String kafkaTopic) {
     factory.removeStorageEngine(storageEngine);
   }
 
+  AbstractStorageEngine openStorageEngine(String storeName) {
```

Comment:
And I am struggling to think if it is better to restore everything inside II process. It looks like a better option in general (for non-DVC case). And we can tweak the logic to claim if a resource is not found in II's metadata management flow, it should by default sent to forked process. (It cannot be in the handover situation). 
For II's metadata management, feel free to check IsolatedIngesitonBackend and let me know if you think otherwise.